src/middlewared/middlewared/etc_files/scst.conf.mako

```diff
@@ -123,6 +131,11 @@ TARGET_DRIVER iscsi {
 %   for access_control in initiator_portal_access:
```

Comment:
Do we know how this behaves if you put a network segment in the webUI? (i.e. `192.168.1.0/24`). Currently, the config will ignore this and write a config entry like so `INITIATOR *\#<host-ip>` The `*` meaning all initiators are allowed to connect.

However, we're writing another entry that includes the `iqn` of the host. Does that mean all initiators are allowed to connect as well as the specific iqn?

Reply:
`auth_network` is not used anywhere in `scst.conf` because
```
# FIXME: Authorized networks for initiators has not been implemented yet, please look for alternatives in SCST
```

Reply:
That's what I'm saying, because we aren't using `auth_network` we already write an `INITIATOR` entry that looks like `*\#<host-ip>`

Your change  will write another `INITIATOR` entry that has `<iqn>\#<host-ip>`. So I'm trying to understand which entry is the correct one? Yours or the `*` entry.

Reply:
@yocalebo good catch! Latest commit fixes this

Reply:
nice!